src/NzbDrone.Core/Datastore/Migration/153_add_on_episodefiledelete_for_upgrade.cs

```diff
@@ -0,0 +1,15 @@
+﻿using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(153)]
+    public class add_on_episodefiledelete_for_upgrade : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Notifications").AddColumn("OnEpisodeFileDeleteForUpgrade").AsBoolean().WithDefaultValue(0);
```

Comment:
Default value should be 1 to maintain existing behaviour.

Reply:
updated